tools/cli.py

```diff
@@ -31,6 +31,7 @@
         'deleteinstance': aws_cli.DeleteInstance,
         'gcstos3': aws_cli.GCSToS3,
         'imageebssnapshottos3': aws_cli.ImageEBSSnapshotToS3,
+        'instanceprofilemitigator': aws_cli.InstanceProfileMitigator,
```

Comment:
I'm not 100% sold on this option name, happy for alternate suggestions.

Reply:
Maybe just `profilemitigator`?